styles/dashboard.module.css

```diff
@@ -30,6 +30,16 @@
     width: 100%;
 }
 
+.hide{
+    font-family: Sora;
```

Comment:
I suggest you to use prettier to format the code correctly automatically (because the syntax is not clean in a lot of places)